File: src/main/java/com/dsm/pick/PickApplication.java
package com.dsm.pick;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
public class PickApplication {

//	@PostConstruct
//	public void setTimeZone() {
//		TimeZone.setDefault(TimeZone.getTimeZone("Asia/Seoul"));
//	}

	public static void main(String[] args) {
		SpringApplication.run(PickApplication.class, args);
	}
}

File: src/main/kotlin/com/dsm/pick/PickApplication.kt
package com.dsm.pick

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching
import springfox.documentation.swagger2.annotations.EnableSwagger2

@EnableSwagger2
@EnableCaching
@SpringBootApplication
class PickApplication

fun main(args: Array<String>) {
    runApplication<PickApplication>(*args)
}
